.vscode/settings.json

```diff
@@ -4,12 +4,11 @@
 
   "editor.renderWhitespace": "all",
 
+  "[php]": {
+    "editor.defaultFormatter": "bmewburn.vscode-intelephense-client"
+  },
+
   // Formatting
   "editor.formatOnSave": true,
-  "phpformatter.composer": true,
-  "phpformatter.arguments": [""],
-
-  // Linting
-  "phpcs.enable": true,
-  "phpcs.standard": "PSR2"
+  "intelephense.format.braces": "psr12"
```

Comment:
I think this is the same, but i'm not positive. I did see some minor bracket motion when saving a file, so i'm not sure.

Reply:
Can you tell me the files that were changed? I am wondering how this change would affect the formatter in our CI. I doubt they support PSR12.

Reply:
This PR didn't change any files directly - stuff would get changed as vscode users format/save things. 

That said, this isn't a great solution. It doesn't seem like a 1:1 replacement and not everyone on the team uses vscode. I wonder if we don't use vscode for formatting at all and lean on `make format` instead, which will be consistent across editors and CI.

In JS and other languages, the formatter and the vscode extension use the same program under the hood, so they always agree. I'm not sure PHP can have the same setup.

Reply:
Let's see if this would break anything in the future. We can anyways fallback on the `make fmt` for formatting.